benchmark_runner/common/oc/oc.py

```
@@ -658,6 +658,24 @@ def apply_security_privileged(self, namespace: str = environment_variables.envir
+    def apply_security_anyuid(self, namespace: str = environment_variables.environment_variables_dict['namespace']):
+        """
+        This method grants anyuid SCC to default service account in namespace, waiting for default SA to exist
+        @param namespace:
+        @return:
+        """
+        last_err = None
+        for _ in range(self.RETRIES):
+            try:
+                self.run(f"{self._cli} get sa default -n {namespace}")
+                return self.run(f"{self._cli} adm policy add-scc-to-user -n {namespace} anyuid -z default")
+            except Exception as err:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)